node/src/main/java/io/airlift/node/NodeInfo.java

```diff
@@ -132,6 +141,21 @@ public NodeInfo(String environment,
         else {
             this.externalAddress = this.internalAddress;
         }
+
+        if (annotationsFile != null) {
+            try {
+                this.annotations = replaceEnvironmentVariables(loadPropertiesFrom(annotationsFile));
```

Comment:
Add `ImmutableMap.copyOf()` since `replaceEnvironmentVariables()` returns a mutable map and we expose it from this class.